modules/odo-key-features.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/understanding-odo.adoc
+
+:_content-type: CONCEPT
+[id="odo-features_{context}"]
+= odo key features
+
+`{odo-title}` is designed to be a simpler and easier to use interface to Kubernetes than `oc`, with the ability to:
```

Comment:
`oc` and `odo` are not targeting the same people. `oc` is for cluster admin, `odo`  for developers. They should be compared in term of audience, not simplicity.

Reply:
I did have:

`{odo-title}` is designed to be simple and easy to use, with the following functionality:


But seemingly the IBM Style Guide does not approve:

IBM SG: "Perceived benefits or characteristics of a product: Avoid phrases such as "simple to use,"
"effortless," "user-friendly," "future-proof," and "easy to understand." Users might not find these
statements true, and inflated language can reduce the credibility of the information."

I will remove the comparison to oc, and try to say it is developer-friendly but in different words